src/Microsoft.Azure.WebJobs.Host/Bindings/Cancellation/CancellationTokenBinding.cs

```diff
@@ -33,7 +33,7 @@ private Task<IValueProvider> BindAsync(CancellationToken token, ValueBindingCont
 
         public Task<IValueProvider> BindAsync(object value, ValueBindingContext context)
         {
-            if (value is CancellationToken)
```

Comment:
It dosen't look like we can use the `as` operator on a struct:
![image](https://user-images.githubusercontent.com/20703505/29110589-68560832-7ce7-11e7-84f3-a6ce8c56d534.png)